entity-framework/core/platforms/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Platforms | Microsoft Docs
+author: rowanmiller
+ms.author: divega
+ms.date: 03/13/2017
+ms.assetid: bfce70e5-7e14-47d3-87b2-e0b93352e955
+ms.technology: entity-framework-core
+uid: core/platforms/index
+---
+
+# Platforms
+
+> [!NOTE]
+> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../ef6/index.md).
+
+We want EF Core to be available anywhere you write .NET code, and we're still working towards that goal. The following table provides guidance for each platform where we want to enable EF Core.
+
+<table>
```

Comment:
Using html as the markdown table syntax doesn't work very well for tables with this much in each cell